src/UseCaseExecutor.ts

```diff
@@ -200,5 +200,20 @@ export class UseCaseExecutor {
     release(): void {
         this._releaseHandlers.forEach(releaseHandler => releaseHandler());
         this._releaseHandlers.length = 0;
+        if (process.env.NODE_ENV !== "production") {
+            this._addWarningNoMoreDispatch();
+        }
+    }
+
+    /**
+     * When child is completed after parent did completed, display warning warning message
+     * @private
+     */
+    private _addWarningNoMoreDispatch() {
```

Comment:
💡 We have already known that `this._parentUseCase` is parent.
If `this._parentUseCase` is released, then call `this._willExecute` in the Child UseCase and throw Warning message.
Is it precision?

Currently approach have a potensial of memory leak.
Because, listen the dead useCase.